clients/client-python/setup.py

```diff
@@ -24,7 +24,7 @@
     python_requires=">=3.8",
     packages=find_packages(exclude=["tests*"]),
     classifiers=[
-        'Development Status :: 3 - Alpha',
+        'Development Status :: 5 - Production/Stable',
```

Comment:
I don't think our current Python client reaches to level 5, why do we update this?

Reply:
OK, I rollback these change.